netbox/dcim/views.py

```diff
@@ -1365,6 +1401,16 @@ class PowerOutletBulkDeleteView(PermissionRequiredMixin, BulkDeleteView):
 # Interfaces
 #
 
+class InterfaceListView(PermissionRequiredMixin, ObjectListView):
+    type = 'Interface'
```

Comment:
Not sure where this came from; is it needed?

Reply:
Can be removed, thanks, will do it shortly.